src/memory-fs.js

```diff
@@ -0,0 +1,199 @@
+const path = require('path');
+
+const { createHash } = require(`crypto`);
+const errors = require('errno');
+const util = require('util');
+const { ReadableStream, WritableStream } = require('stream');
+const MemoryFileSystemError = require('memory-fs/lib/MemoryFileSystem');
+
+const debug = util.debuglog('astroturf:memory-fs');
+
+const returnsTrue = () => true;
+const returnsFalse = () => false;
+
+const md5 = input => {
```

Comment:
@taion is this worth doing vs storing the content and doing  a straight string compare?

Reply:
actually you could just do `Buffer.compare`

Reply:
oo